src/ir_Sharp.cpp

```diff
@@ -375,11 +388,10 @@ uint8_t IRSharpAc::getMode(void) {
 }
 
 void IRSharpAc::setMode(const uint8_t mode) {
-  setBit(&remote[kSharpAcBytePower], kSharpAcBitModeNonAutoOffset,
-         mode != kSharpAcAuto);
   switch (mode) {
     case kSharpAcAuto:
     case kSharpAcDry:
+      this->setFan(2); // When Dry or Auto, Fan always 2(Auto)
```

Comment:
`src/ir_Sharp.cpp:394:  At least two spaces is best between code and comments  [whitespace/comments] [2]`